package/chalicelib/awsAuditor.py

```diff
@@ -15,7 +15,7 @@ def main():
 
```

Comment:
It looks like this file is not up to date with master. Please make sure this and other files are sync'ed up.

Reply:
done